build/importer/Dockerfile

```diff
@@ -1,16 +1,19 @@
-FROM alpine:3.3
+FROM fedora:28
 RUN mkdir /data
 ARG entrypoint
 ARG runArgs
 ARG depFile1
 ARG depFile2
 ENV RUN_ARGS=$runArgs
+ENV GIMME_GO_VERSION=1.10
 COPY $depFile1 /
 COPY $depFile2 /
 COPY ./$entrypoint ./run-binary
-RUN apk update && apk add qemu-img \
-  xz \
-  gzip
-RUN apk add ca-certificates && rm -rf /var/cache/apk/*
+RUN dnf install -y qemu-img xz gzip
+RUN mkdir -p /gimme && curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | HOME=/gimme bash >> /etc/profile.d/gimme.sh
+ENV GOPATH="/go" GOBIN="/usr/bin"
+RUN \
+    mkdir -p /go && \
+    source /etc/profile.d/gimme.sh
```

Comment:
should `-v=` be an arg to ENTRYPOINT below?

Reply:
Right now, there is nothing written to build the controller or importer images, so that's TBD.  I'm  going to simplify their images into something a little more flexible.